test/opexecuter.cpp

```diff
@@ -89,17 +89,15 @@ void OpExecuter::ValidateGraph(const Graph& graph,
                                const vector<string> allowed_nodes) {
   NGRAPH_VLOG(5) << "Validate graph";
   bool found_test_op = false;
-  Node* test_op;
   for (Node* node : graph.nodes()) {
     if (node->IsSource() || node->IsSink()) {
       continue;
     } else if (node->type_string() == test_op_type_) {
       // only one node of type test_op
       ASSERT_FALSE(found_test_op) << "Only one op of type " << test_op_type_
                                   << " should exist in the graph. Found nodes "
-                                  << node->name() << " and " << test_op->name();
+                                  << node->name() << " and " << node->name();
```

Comment:
The original version will print the duplicated node names, but after the change it will print one of those node names twice. If we are removing the `test_op`, we probably should rephrase this message to be sth like "Found duplicated type nodes for"<< node->name();

Reply:
Yes - you are right. We are already printing the node->name() - so not sure why I didn't delete the second occurrence. The real issue with the original was an error of attempting to read a variable which may not have been initialized (which would be the case if the node type is a test op). This was genuinely flagged by the compiler warning - so I was removing this. Will clean this up.